Resources/Prototypes/Entities/Mobs/Species/shadowkin.yml

```diff
@@ -1,6 +1,7 @@
 - type: entity
   save: false
   parent:
+    - BaseMobSpeciesOrganic
     - MobBloodstream
```

Comment:
Since they're parented to BaseMobSpeciesOrganic, you can remove all other parents.